packages/core/src/components/TextField/__snapshots__/TextField.test.tsx.snap

```diff
@@ -6174,8 +11245,8 @@ exports[`TextField with outlined variant should render error state properly  1`]
   -ms-flex-direction: row;
   flex-direction: row;
   box-sizing: border-box;
-  height: 5.6rem;
-  padding: 0 1.6rem;
+  height: NaNrem;
```

Comment:
Seems like something is going wrong here...Let me look into why we're getting NaN for this one and get back to you.

Reply:
I need to open a separate PR for a couple of changes that Travis requested. I'll address this issue there. Otherwise, looks good.